HW04/aleksei_zhuchkov/spec/mentor_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../lib/mentor'
+require_relative '../lib/notification'
+require_relative '../lib/student'
+require_relative '../lib/homework'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'student_name', surname: 'student_surname') }
+  let(:mentor) { described_class.new(name: 'mentor_name', surname: 'mentor_surname') }
+  let(:homework) do
+    mentor.add_homework(title: 'test_title',
+                        description: 'test_description',
+                        student: student)
+  end
+
+  describe 'check method add_homework' do
+    it 'add homework to student to be a instance of class Homework' do
+      expect(homework).to be_an_instance_of(Homework)
+    end
+
+    it 'check right value homework description' do
+      expect(homework.description).to eq('test_description')
+    end
+
+    it 'check right value homework title' do
+      expect(homework.title).to eq('test_title')
+    end
+  end
+
+  describe 'check method subscride' do
+    it 'return value is instance of class Array' do
+      expect(mentor.subscribe_to!(student)).to be_an_instance_of(Array)
+    end
+
+    it 'storage can\'t be empy when mentor subsribe to student' do
```

Comment:
read about it block and what should be written in it, please

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/52de5fc33804a43b51fef47f2e218228398f308d